test/uk/gov/hmrc/tai/controllers/TaxAccountSummaryControllerSpec.scala

```diff
@@ -63,66 +63,6 @@ class TaxAccountSummaryControllerSpec extends PlaySpec with MockitoSugar with Np
       }
     }
 
```

Comment:
Could replace these tests with a single test that verifies any exception that isn't a BadRequest or NotFoundException still escapes